relude.cabal

```diff
@@ -104,7 +104,7 @@ source-repository head
   location: git@github.com:kowainik/relude.git
```

Comment:
Can you also please add GHC-9.2 to the `tested-with` above? 🙂

Reply:
Good catch, should be fixed by 23feee16dae03e0796ac3cb2e318c943d906ca22